database/sqlite/sqlite_functions.h

```diff
@@ -24,6 +24,7 @@ typedef enum db_check_action_type {
     DB_CHECK_CONT = 0x00008
 } db_check_action_type_t;
 
+#define SQL_MAX_RETRY (100)
```

Comment:
100 retries seems a little bit excessive to me because each retry sleeps for 50ms which means that the maximum duration will be 5 seconds.